src/main/java/org/fcrepo/camel/processor/SparqlDeleteProcessor.java

```diff
@@ -40,7 +40,8 @@ public void process(final Exchange exchange) throws IOException {
         final Message in = exchange.getIn();
         final String subject = ProcessorUtils.getSubjectUri(in);
 
-        in.setBody("DELETE WHERE { <" + subject + "> ?p ?o }");
+        in.setBody("DELETE WHERE { <" + subject + "> ?p ?o };\n" +
```

Comment:
This looks like a (valid) hack for plucking out specific triples. If so, please add a comment to the effect of why you are requiring the fcr:export path element. There is a good chance that the jcr/xml export may be going away at some point, and it is important that we know if this line of code is a true dependency.